src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -129,10 +129,31 @@ const getBackToSignInLink = (settings) => {
   ];
 };
 
+const getSignUpLink = (appState, settings) => {
+  const signupLink = [];
+
+  if (appState.hasRemediationObject(RemediationForms.SELECT_ENROLL_PROFILE)) {
+    const signupLinkData = {
+      'type': 'link',
+      'label': loc('signup', 'login'),
+      'name': 'enroll'
+    };
+    if (_.isFunction(settings.get('registration.click'))) {
+      signupLinkData.click = settings.get('registration.click');
```

Comment:
nit: consider renaming this to `signupLinkData.registrationClickHandler` or `signupLinkData.clickHandler`